### DISCUSSION TITLE
可以增加基金的起始及结束日期吗？

### DISCUSSION THREAD

**Discussion by lzjkevin** [Micro-sheep/efinance#88] [Ideas]
可以增加基金的起始及结束日期吗？

**Comment by Micro-sheep** [comment]
> 可以增加基金的起始及结束日期吗？

这个没法加，目前提供了一个叫 pz 的参数来控制需要获取的数据的最新条数，如果不想获取全部，你可以更改它，然后对返回的结果，根据日期做筛选。

**Reply by lzjkevin** [reply]
thanks for update
